grid_map_ros/src/GridMapRosConverter.cpp

```diff
@@ -74,21 +95,26 @@ bool GridMapRosConverter::fromMessage(const grid_map_msgs::GridMap& message, gri
   return true;
 }
 
-bool GridMapRosConverter::fromMessage(const grid_map_msgs::GridMap& message, grid_map::GridMap& gridMap)
+bool GridMapRosConverter::fromMessage(
+  const grid_map_msgs::msg::GridMap & message,
+  grid_map::GridMap & gridMap)
 {
   return fromMessage(message, gridMap, std::vector<std::string>(), true, true);
 }
 
-void GridMapRosConverter::toMessage(const grid_map::GridMap& gridMap, grid_map_msgs::GridMap& message)
+void GridMapRosConverter::toMessage(
+  const grid_map::GridMap & gridMap,
+  grid_map_msgs::msg::GridMap & message)
 {
   toMessage(gridMap, gridMap.getLayers(), message);
 }
 
-void GridMapRosConverter::toMessage(const grid_map::GridMap& gridMap, const std::vector<std::string>& layers,
-                      grid_map_msgs::GridMap& message)
+void GridMapRosConverter::toMessage(
+  const grid_map::GridMap & gridMap, const std::vector<std::string> & layers,
+  grid_map_msgs::msg::GridMap & message)
 {
-  message.info.header.stamp.fromNSec(gridMap.getTimestamp());
-  message.info.header.frame_id = gridMap.getFrameId();
+  message.header.stamp = rclcpp::Time(gridMap.getTimestamp());
```

Comment:
is `getTimestamp()` returning time in nanoseconds? The rclcpp::Time() input is nanoseconds (or the other API is seconds, nanoseconds)

Reply:
Just overall do a once over and make sure the times are correct

Reply:
I guess should be fine, it return is in nanoseconds: https://github.com/ANYbotics/grid_map/blob/02c5874deddfbf038241e6dc2438ef7b33f382ef/grid_map_core/include/grid_map_core/GridMap.hpp#L410